tasks/Create-A-wall.md

```diff
@@ -0,0 +1,27 @@
+#Labyrinth: Create a Wall
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+We can add new features to the labyrinth which improve the game.
```

Comment:
We can use your issue statement here: 
"As most of the tiles have the same walls and with same walls in every tile the game will look monotonous, please add more walls!"